src/test/README.md

```diff
@@ -12,22 +12,30 @@ that runs all of the unit tests. The main source file for the test library is fo
 
 ### Compiling/running unit tests
 
-Unit tests will be automatically compiled if dependencies were met in `./configure`
-and tests weren't explicitly disabled.
+Unit tests will be automatically compiled if the dependencies are met during the CMake configuration process
+and the tests haven't been explicitly disabled.
 
-After configuring, they can be run with `make check`, which includes unit tests from
-subtrees, or `make && make -C src check-unit` for just the unit tests.
+To configure the project, build it in the 'build' directory, and then run all unit tests using 8 threads,
```

Comment:
Not sure if adding parallelism options is related to the migration to CMake. In any case, explicitly mentioning a fixed number of threads gives the impression that this number is a hard limit and that the option itself is obligatory.

If you insist on mentioning the `-j` option, you could follow the pattern from the build notes.

Reply:
Done